libs/vm_modules/examples/commodity_prediction/script/commodity_prediction.etch

```diff
@@ -0,0 +1,64 @@
+function main()
+
+if (System.Argc() != 2)
+  print("Usage: VM SCRIPT_FILE PATH/TO/model_files/ /PATH/TO/INPUT/file.csv");
+  return;
+endif
+
+var loader = CommodityDataLoader();
+loader.AddData(System.Argv(1));
+
+//print(loader.GetNext().ToString());
+
+// mkae graph with correct architecture
+var graph = Graph();
+
+graph.AddPlaceholder("num_input");
+graph.AddDropout("dropout_0", "num_input", 1.0f);
+graph.AddFullyConnected("hidden_dense_1", "dropout_0", 118, 216);
+graph.AddSoftmax("softmax_1", "hidden_dense_1");
+graph.AddDropout("dropout_1", "softmax_1", 1.0f);
+graph.AddFullyConnected("hidden_dense_2", "dropout_1", 216, 108);
+graph.AddSoftmax("softmax_2", "hidden_dense_2");
+graph.AddDropout("dropout_2", "softmax_2", 1.0f);
+graph.AddFullyConnected("output_dense", "dropout_2", 108, 54);
+graph.AddSoftmax("softmax_3", "output_dense");
+
+
+// read in weights
+var file_weights0 = System.Argv(0) +  "/output/keras_h7_aluminium_px_last_us/model_weights/hidden_dense_1/hidden_dense_1_12/kernel:0.csv";
```

Comment:
lets break this work into a few separate etch functions for clarity. e.g:

function build_graph()
...
endfunction

function read_weights()
...
endfunction

function main()
build_graph();
read_weights();
...
endfunction